tests/providers/test_bank.py

```diff
@@ -16,6 +16,19 @@ def test_bban(self):
         bban = self.factory.bban()
         assert re.match(r"\d{11}", bban)
 
+class TestFiFi(unittest.TestCase):
```

Comment:
same here re: blank lines: you need 1 more above and one more below

Reply:
Should be OK now, pep8 reports no problems on the test_bank.py